test/integration/user_follows_place_test.rb

```diff
@@ -0,0 +1,40 @@
+require "test_helper"
+class UserFollowsPlaceSpec < ActionDispatch::IntegrationTest
+  include Capybara::DSL
+  require 'vcr'
+  require 'webmock'
+
+  test "follows a place" do
+    skip
+    Capybara.current_driver = Capybara.javascript_driver
+    VCR.use_cassette("follow") do
+      login_user
+      visit "/"
+      page.fill_in 'nav-search', 
+        :with => 'Turing School of Software & Design'
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.